src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx

```diff
@@ -0,0 +1,151 @@
+    this.setState({
+      showingMoreArtists: !this.state.showingMoreArtists,
+    })
+  }
+
+  renderSingleArtist(artist: Artist) {
+    return (
+      <React.Fragment>
+        <Flex flexDirection="row">
+          {this.renderArtistName(artist.name, artist.href)}
+          <Sans color={color("black60")} size="6" mx={1}>
+            &middot;
+          </Sans>
+          <Sans color={color("black60")} size="4">
+            Follow
+          </Sans>
+        </Flex>
+      </React.Fragment>
+    )
+  }
+
+  renderArtistName(artistName: string, href: string) {
+    return href ? (
+      <>
+        <TouchableWithoutFeedback onPress={this.handleTap.bind(this, href)}>
+          <Serif size="5t" weight="semibold">
+            {artistName}
+          </Serif>
+        </TouchableWithoutFeedback>
+      </>
+    ) : (
+      <>
+        <Serif size="5t" weight="semibold">
+          {artistName}
+        </Serif>
+      </>
+    )
+  }
+
+  renderMultipleArtists = () => {
+    const {
+      artwork: { artists },
+    } = this.props
+
+    const truncatedArtists = !this.state.showingMoreArtists ? artists.slice(0, 3) : artists
+    const artistNames = truncatedArtists.map((artist, index) => {
+      const artistNameWithComma = index !== artists.length - 1 ? artist.name + ", " : artist.name
+      return (
+        <React.Fragment key={artist.href}>{this.renderArtistName(artistNameWithComma, artist.href)}</React.Fragment>
+      )
+    })
+
+    return (
+      <Flex flexDirection="row" flexWrap="wrap">
+        {artistNames}
+        {!this.state.showingMoreArtists &&
+          artists.length > 3 && (
+            <TouchableWithoutFeedback onPress={this.showMoreArtists}>
+              <Serif size="5t" weight="semibold">
+                {artists.length - 3} more
+              </Serif>
+            </TouchableWithoutFeedback>
+          )}
+      </Flex>
+    )
+  }
+
+  render() {
+    const { artwork } = this.props
+    return (
+      <Box textAlign="left">
+        <Flex flexDirection="row" flexWrap="wrap">
+          {artwork.artists.length === 1 ? this.renderSingleArtist(artwork.artists[0]) : this.renderMultipleArtists()}
+          {artwork.artists.length === 0 &&
+            artwork.cultural_maker &&
+            this.renderArtistName(artwork.cultural_maker, null)}
+        </Flex>
+        <Serif color={color("black60")} size="3t" m="0" p="0">
+          {artwork.title}, {artwork.date}
+        </Serif>
+        <Serif color={color("black60")} size="3t">
+          {artwork.medium}
+        </Serif>
+        <Serif color={color("black60")} size="3t">
+          {Constants.CurrentLocale === "en_US" ? artwork.dimensions.in : artwork.dimensions.cm}
+        </Serif>
+        {artwork.edition_of && (
+          <Serif color={color("black60")} size="3t">
+            {artwork.edition_of}
+          </Serif>
+        )}
+        {artwork.attribution_class && (
+          <Serif color={color("black60")} size="3t">
```

Comment:
Shouldn't need to use the `color` helper everywhere (all theme values should be accessible as strings / numbers). At the top of the react tree can wrap with 

```tsx
import { Theme } from '@artsy/palette'
...
<Theme>
  <App />
</Theme>
```
Which then injects the styled-system context from which palette pulls its values from.

Reply:
I tried that but it doesn't seem to be working for me

Reply:
```Unhandled JS Exception: Invariant Violation: Invalid prop `color` supplied to `StyleSheet generated`: black60
Valid color formats are
  - '#f0f' (#rgb)
  - '#f0fc' (#rgba)
  - '#ff00ff' (#rrggbb)
  - '#ff00ff00' (#rrggbbaa)
  - 'rgb(255, 255, 255)'
  - 'rgba(255, 255, 255, 1.0)'
  - 'hsl(360, 100%, 100%)'
  - 'hsla(360, 100%, 100%, 1.0)'
  - 'transparent'
  - 'red'
  - 0xff00ff00 (0xrrggbbaa)```

Reply:
Ah, interesting -- I'm facing something similar in a separate issue, which I thought was related to a styled-components upgrade. It's validating values :(

Reply:
Nevermind! Just needed to rebase

Reply:
It actually works ok its just the storybook for the component that gets the error because it doesn't have access to Theme. The storybook for the whole screen works fine.

Reply:
Can you try adding this to the top of the story really quick?

```js
storiesOf(...).addDecorator(storyFn => {
    return <Theme>{storyFn()}</Theme>
  })
.add(...) 
```

(the add decorator bit)

Does it still error out without the `color` function?

Reply:
(As an aside, how very exciting that this sometimes works -- it means that the styled-components upgrade on my side isn't as catastrophic as it seems...)

Reply:
I ended up just enclosing them all in Theme but I'll try that instead

Reply:
It works!

Reply:
In reaction we use our own [custom `storiesOf` function](https://github.com/artsy/reaction/blob/master/src/__stories__/storiesOf.js). It's specific to Reaction but something similar could be devised for Emission that provides the `<Theme>`. 

Would suggest setting that up in Emission at some point (in a separate PR) so that y'all don't need to use `addDecorator` everywhere.  cc @ds300

Reply:
pretty sure storybook decorators can be configured globally. See e.g. https://storybook.js.org/docs/basics/writing-stories/#using-decorators

Reply:
Can also leave that for another ticket though.

Reply:
Oh wow, wasn't aware of the global option 👍